docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,213 @@
+---
+slug: quickstart
+append_help_link: true
+title: Quickstart
+hide_title: false
+description: Learn how to set up Semgrep and scan your first repository.
+tags:
+  - quickstart
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Learn how to set up Semgrep, scan your first project for security issues, and view your findings.
+
+:::info prerequisites
+You must have Python 3.8 or later installed on the machine where the Semgrep CLI is running.
+:::
+
+1. Navigate to [Semgrep Cloud Platform (SCP)](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to [grant Semgrep the needed permissions](/semgrep-cloud-platform/getting-started/#requested-permissions-for-github-and-gitlab) and proceed.
+
+2. Provide the **Organization display name** you'd like to use, then click **Create new organization**.
+
+3. Launch your CLI, and follow the instructions on the [**Scan a project on your machine**](https://semgrep.dev/onboarding/scan) page. For your convenience, the same information is presented below, along with instructions for Windows users.
+
+    <Tabs
+        defaultValue="macOS"
+        values={[
+        {label: 'macOS', value: 'macOS'},
+        {label: 'Linux', value: 'Linux'},
+        {label: 'Windows Subsystem for Linux (WSL)', value: 'Windows Subsystem for Linux (WSL)'},
+        {label: 'Docker', value: 'Docker'},
+        ]}
+    >
+
+    <TabItem value='macOS'>
+
+    1. Install the Semgrep CLI and confirm the installation:
+
+        ```console
+        # with homebrew
+        brew install semgrep
+
+        # with pip
+        python3 -m pip install semgrep
+
+        # confirm
+        semgrep --version
+        ```
+
+        :::note
+        **Homebrew users:** ensure that you've [added Homebrew to your PATH](https://docs.brew.sh/FAQ#my-mac-apps-dont-find-homebrew-utilities).
+        :::
+
+    2. Log in to your Semgrep account. Running this command launches a browser window, but you can also use the link that's returned in the CLI to proceed:
+
+        ```console
+        semgrep login
+        ```
+
+    3. In the **Semgrep CLI login**, click **Activate** to proceed.
+
+    4. Return to the CLI, navigate to the root of your repository, and run your first scan:
+
+        ```console
+        semgrep ci
+
+        # Alternatively, run an offline-only SAST scans; no SCA scan is performed
```

Comment:
```suggestion
            # Alternatively, run an offline-only SAST scan; no SCA scan is performed
```

I see that you put it here, but it sort of negates the whole flow because the user doesn't have to sign in for this. I apologize - maybe better to revert in keeping in line with product.